agent.exs

```diff
@@ -1,47 +1,47 @@
 defmodule Appsignal.Agent do
-  def version, do: "1d8917f"
+  def version, do: "f6353b4"
```

Comment:
That's not the version in which the transaction debug mode is: https://github.com/appsignal/appsignal-agent/releases/tag/v-c348132